Gems/AzQtComponentsForPython/Code/Platform/Windows/PAL_windows.cmake

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(PAL_TRAIT_AZQTCOMPONENTSFORPYTHON_SUPPORTED TRUE)
+set(PAL_TRAIT_AZQTCOMPONENTSFORPYTHON_TEST_SUPPORTED FALSE)
+set(PAL_TRAIT_AZQTCOMPONENTSFORPYTHON_EDITOR_TEST_SUPPORTED FALSE)
```

Comment:
Is there any platform for which the tests ARE supported?  If not, why have them?

Reply:
Should be gone, fixed.